docs/interfaces/paypalcommercepaymentinitializeoptions.md

```diff
@@ -5,6 +5,7 @@
 A set of options that are required to initialize the PayPal Commerce payment
 method for presenting its PayPal button.
 
+Please note that the minimum version of checkout-sdk is 1.100
```

Comment:
This file is auto-generated. You shouldn't edit it directly as it'd get overridden when we regenerate the doc. Instead, you should edit this file instead - https://github.com/bigcommerce/checkout-sdk-js/blob/master/src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts#L8. We extract the comment and re-generate this markdown file every time we cut a new release.